src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16910.cs

```diff
@@ -28,6 +28,7 @@ public void BindingUpdatesFromProgrammaticRefresh()
 	}
 #if TEST_FAILS_ON_CATALYST //Scroll actions cannot be performed on the macOS test server
 	[Test]
+ 	[FlakyTest("Issue to reenable this test: https://github.com/dotnet/maui/issues/28178")]
```

Comment:
I think it should be fixed by https://github.com/dotnet/maui/pull/28137. Could we merge it, and if still fails sometimes mark it as Flaky?